src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -2804,14 +2804,17 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="RepairOperationNotSupported" xml:space="preserve">
     <value>The installer technology in use does not support repair.</value>
   </data>
-    <data name="NoAdminRepairForUserScopePackage" xml:space="preserve">
+  <data name="NoAdminRepairForUserScopePackage" xml:space="preserve">
     <value>The package installed for user scope cannot be repaired when running with administrator privileges.</value>
   </data>
   <data name="APPINSTALLER_CLI_ERROR_ADMIN_CONTEXT_REPAIR_PROHIBITED" xml:space="preserve">
     <value>Repair operations involving administrator privileges are not permitted on packages installed within the user scope.</value>
-  </data>  
+  </data>
   <data name="RepairFailedWithCode" xml:space="preserve">
     <value>Repair failed with exit code: {0}</value>
     <comment>{Locked="{0}"} Error message displayed when an attempt to repair an application package fails. {0} is a placeholder replaced by an error code.</comment>
   </data>
+  <data name="APPINSTALLER_CLI_ERROR_SQLITE_CONNECTION_TERMINATED" xml:space="preserve">
+    <value>The SQLite connection was terminated to prevent corruption.</value>

```

Comment:
We may need to trigger a new localization update for this.

Reply:
I'm not sure that the error strings really necessitate that, although they can potentially be show to users not using the `error` command if the exception makes to the root.  They were not localized for a very long time though, and I never knew of anyone complaining.

So I wouldn't hold up any builds based on it.